### PR TITLE
add rules of hooks for eslint

### PR DESCRIPTION
**Pull request by valpinkman**
also, removed all unnecessary Fragments

### CODE REVIEW

**Review by juan-cortes — APPROVED** [review]

**Comment by valpinkman** [comment]
@gre wanted to check the changes I think

File: src/renderer/families/tezos/DelegateFlowModal/steps/StepVerification.js [hunk 1]
```diff
@@ -20,9 +20,11 @@ const StepVerification = ({
   }, [signTransaction, transitionTo]);
 
   // didMount
+  // this was done only on mount on v1, so I removed the
+  // dependencies to simulate that (val)
```
Comment (gre) [inline src/renderer/families/tezos/DelegateFlowModal/steps/StepVerification.js:24]: (interesting to dig why this code is needed. I thought we had "useSignTransaction" for this)

**Review by gre — CHANGES_REQUESTED** [review]

File: src/renderer/Default.js [hunk 2]
```diff
@@ -69,7 +69,7 @@ const Default = () => {
     if (ref && ref.current) {
       ref.current.scrollTo(0, 0);
     }
-  }, [location]);
+  }, [location, ref]);
```
[the remark this answers is not in the document]
Comment (valpinkman) [reply src/renderer/Default.js]: will fix

File: src/renderer/families/tezos/DelegateFlowModal/steps/StepVerification.js (hunk 1 above)
Comment (valpinkman) [reply src/renderer/families/tezos/DelegateFlowModal/steps/StepVerification.js:24]: we can probably use `useSignTransaction`

Comment (gre) [reply src/renderer/families/tezos/DelegateFlowModal/steps/StepVerification.js:24]: maybe it's diff level of things. we'll rediscuss in future

**Review by gre — APPROVED** [review]

**Pull request merged**